webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -60,6 +76,62 @@ const ContentViewRepositories = ({ cvId }) => {
   ];
   const loading = status === STATUS.PENDING;
 
+  const toggleBulkAction = () => {
+    setBulkActionOpen(!bulkActionOpen);
+  };
+  const onAdd = (repos) => {
+    const { repository_ids: repositoryIds } = details;
+    dispatch(updateContentView(cvId, { repository_ids: repositoryIds.concat(repos) }));
+  };
+
+  const onRemove = (repos) => {
+    const reposToDelete = [].concat(repos);
+    const { repository_ids: repositoryIds } = details;
+    const deletedRepos = repositoryIds.filter(x => !reposToDelete.includes(x));
```

Comment:
naming nitpick - is this the deleted repos, or the repos that should remain in the CV? If the latter, can we call it something like `remainingRepos` or `stillAddedRepos` (or another suggestion)?